shared/common-adapters/tab-bar.native.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+import React, {Component} from 'react'
+import {TouchableWithoutFeedback} from 'react-native'
+import {globalStyles, globalColors} from '../styles/style-guide'
+import Box from './box'
+import Text from './text'
+
+import type {Props, ItemProps} from './tab-bar'
+
+class TabBarItem extends Component {
+  props: ItemProps;
+
+  render () {
+    return this.props.children
+  }
+}
+
+class TabBar extends Component {
+  props: Props;
+  static Item: Class<TabBarItem>;
+
+  _labels (): Array<React$Element> {
+    const tabWidth = this.props.tabWidth || 93
+
+    return (this.props.children || []).map(item => (
```

Comment:
mmm, lemme see, we should be able to return nulls, we shoudln't need empty boxes